Fundamentals/Setup/Requirements/index-v11.md

```diff
@@ -0,0 +1,66 @@
+---
+versionFrom: 11.0.0
+---
+
+# Minimum System Requirements
+
+## Browsers
+
+The Umbraco UI works in all modern browsers:
+
+* Chrome (Latest)
+* Edge (Chromium)
+* Firefox (Latest)
+* Safari (Latest)
+
+## Local Development
+
+* Either OS:
+  * Microsoft Windows 7 SP1, 8.1, 10 and 11
+  * MacOS High Sierra 10.13
+  * Linux (Ubuntu, Alpine, CentOS, Debian, Fedora, openSUSE and other major distributions)
+* One of the following .NET Tools or Editors:
+  * [Visual Studio Code](https://code.visualstudio.com/) with the [IISExpress extension](https://marketplace.visualstudio.com/items?itemName=warren-buckley.iis-express)
+  * [Microsoft Visual Studio](https://www.visualstudio.com/) 2022
```

Comment:
v17.4 minimum is required for .net7 RC2 - currently, this is available only on preview edition